code/index.html

```diff
@@ -3,28 +3,131 @@
+      <h2 id ="offer">what we offer.</h2>
+      <p>
+        Lsed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquya, 
+        sed diam voluptua. At vero eos et accusam et ju
+        no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. 
+      </p>  
+        <div>
+        <h3 id="service-header-1" class="service-header">people</h3>
+          <p id="service-content-1" class="service-content">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, 
+            sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, 
+            no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet.
+          </p>
+        </div>
+
+        <div>
+        <h3 id="service-header-2" class="service-header">fashion</h3>
+          <p id="service-content-2" class="service-content">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, 
+            sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, 
+            no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet.
+          </p>
+        </div>
+  
+        <div>
+        <h3 id="service-header-3" class="service-header">events</h3>
+          <p id="service-content-3" class="service-content">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, 
+            sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren,               no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet.
+            </p>
+          </div>
+  
+        <div>
+        <h3 id="service-header-4" class="service-header">others</h3>
+          <p id="service-content-4" class="service-content">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, 
+            sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, 
+            no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet.
+            </p>
+        </div>
+    </section>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+    <section class="formbox">
+      <h2 id="contact">let's get in touch.</h2>
+      <form action="http://httpbin.org/anything" method="POST">
+        <div class="formelement-personal">
+          <div><label for="fname" class="information-type">name</label>
+            <input type="text" id="name" name=name class="labelbox" required><br>
```

Comment:
This input is not connected with label. As you know, if we want to bind them, we need to establish `for - id` attribute connection. Both `for` and `id` attributes needs to have same value :

```
<label for="fname" class="information-type">name</label>
<input type="text" id="fname" name="name" class="labelbox" required>
```

Moreover, let's delete `<br />` tags, they are deprecated